src/cachingreader.cpp

```diff
@@ -123,18 +123,6 @@ CachingReaderChunkForOwner* CachingReader::allocateChunk(SINT chunkIndex) {
     //qDebug() << "Allocating chunk" << pChunk << pChunk->getIndex();
     m_allocatedCachingReaderChunks.insert(chunkIndex, pChunk);
 
-    // Adjust the least-recently-used item before inserting the
-    // chunk as the new most-recently-used item.
-    if (nullptr == m_lruCachingReaderChunk) {
-        if (nullptr == m_mruCachingReaderChunk) {
-            m_lruCachingReaderChunk = pChunk;
-        } else {
-            m_lruCachingReaderChunk = m_mruCachingReaderChunk;
-        }
-    }
-    // Insert the chunk as the new most-recently-used item.
-    pChunk->insertIntoListBefore(m_mruCachingReaderChunk);
```

Comment:
I did not got the point why it is OK, not to add the new chunk to the linked list.
Is it sure, that the chunk ends finally in th list? 
I am sure it is, but I don't see it.